DESCRIPTION

```diff
@@ -3,8 +3,12 @@ Type: Package
 Title: Dirichlet-Multinomial Mixture Model Machine Learning for
         Microbiome Data
 Version: 1.39.0
-Author: Martin Morgan <martin.morgan@roswellpark.org>
-Maintainer: Martin Morgan <martin.morgan@roswellpark.org>
+Authors@R: c(
+          person("Martin", "Morgan", role = "aut"),
+          person("Khadijah", "Amusat", role = "ctb"),
+                  comment = "Converted DirichletMultinomial vignette from Sweave to RMarkdown / HTML."),
+          person("Bioconductor Package Maintainer", role = "cre",
+                email = "maintainer@bioconductor.org"))          
```

Comment:
- [ ] Remove lines 10-11 since Martin is the maintainer.
- [ ] Replace the comma at the end of line 9 with a closing parenthesis.